### PR TITLE
Reorganize namespace to be consistent with tbdex-js

### PR DESCRIPTION
**Pull request by KendallWeihe**
Throwing this out there, embrace debate. I like the namespacing of the `tbdex-js` package and would be nice to be consistent across both. I recognize they're different languages so it isn't a 1:1 translation, but the more overlap the merrier, right?

Also, double check my Kotlin things cause I'm a total noob.

How do we feel about `website.tbd` versus `xyz.block`?

### CODE REVIEW

File: protocol/src/main/kotlin/website/tbd/tbdex/protocol/Message.kt [hunk 1]
```diff
@@ -19,7 +21,7 @@ class MessageMetadata(
   val createdAt: OffsetDateTime
 )
 
-sealed interface MessageData
```
Comment (KendallWeihe) [inline protocol/src/main/kotlin/website/tbd/tbdex/protocol/Message.kt:22]: This was giving me an error so I just unsealed it... hoping that's acceptable?

Comment (phoebe-lew) [reply protocol/src/main/kotlin/website/tbd/tbdex/protocol/Message.kt:22]: oh yeah I know what the issue is. fine for now. for sealed classes/interfaces all the implementers need to be in the same file

**Comment by ethan-tbd** [comment]
> How do we feel about `website.tbd` versus `xyz.block`?

What's the reason for using these namespaces? Unsure because `protocol/src/main/kotlin/website/tbd/tbdex/protocol/...` seems pretty wordy?

**Comment by KendallWeihe** [comment]
> > > How do we feel about `website.tbd` versus `xyz.block`?
> > 
> > 
> > What's the reason for using these namespaces? Unsure because `protocol/src/main/kotlin/website/tbd/tbdex/protocol/...` seems pretty wordy?
> 
> Yeah @KendallWeihe I think we could go `tbdex.sdk.<package-name>` or `tbdex.<package-name>` to be more consistent? Unless you had other reasons for including `website.tbd`

I'm cool with just `tbdex.*` (no need to include `sdk` as that's implied), as I understand it, the Java pattern is to use the org's host name... but now that I think about it, this is OSE so that doesn't apply (at least not to the same degree)

We can move this forward concept, but need to do it at a time where things aren't changing to such a large degree. Since we  added a bunch of changes to `plew.add-tests` after branching, and then subsequently [merged into main](https://github.com/TBD54566975/tbdex-kt/pull/2), this PR is no longer worth moving forward. Closing for now, we can try again in the future.

**Pull request closed without merging**